msbuild/Xamarin.iOS.Tasks/Tasks/CompileEntitlements.cs

```diff
@@ -1,6 +1,46 @@
+using System.Collections.Generic;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
 using Xamarin.MacDev.Tasks;
-namespace Xamarin.iOS.Tasks {
-	public class CompileEntitlements : CompileEntitlementsTaskBase {
```

Comment:
formatting changes

Reply:
I think this change is correct or at least consistent with the rest of the tasks, right?

I think I need to fix the `DefaultEntitlementsPath` which is not using this format

Reply:
ideally there would not be any formatting changes inside PR (if really needed this should be a separate PR)
This helps merging and/or reverting, save time reviewing PR and makes `git blame` more useful 😄

Reply:
Makes sense, I just fixed this